Meta/CMake/commonmark_spec.cmake

```diff
@@ -1,9 +0,0 @@
-option(ENABLE_COMMONMARK_SPEC_DOWNLOAD "Enable download of commonmark test suite at build time" ON)
```

Comment:
please remove documentation references and CI references to ENABLE_COMMONMARK_SPEC_DOWNLOAD in this commit

Reply:
It is not mentioned anywhere